views/error.html

```diff
@@ -60,6 +67,10 @@
       </div>
     </section>
 
+      <script id="homepage-script"
+          src="/scripts/vendor/require.min.js"
+          data-main="/{{ localeInfo.locale }}/error/scripts/main.js">
+      </script>
```

Comment:
You aren't loading a `requirejs` module, so I think this can just be:

```
<script src="/{{ localeInfo.locale }}/error/scripts/main.js"></script>
```